catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -584,6 +585,10 @@ public Serializable parseDate(Serializable value) {
       return null;
     }
 
+    if (StringUtils.isEmpty(value)) {
```

Comment:
✏️ nit: could combine this check with the check on 584. It's just style though, some prefer to be more explicit. Approved either way.

```
if (!(value instanceof String) || StringUtils.isEmpty(value)) {
      return null;
    }
```

Reply:
I'm biased for this code 😂 but I do for this particular case like the explicit separate statements. (primarily because of the inverted first conditional - with that plus adding in an OR i think it adds a nontrivial amount of processing needed to read/process this code)